modules/@apostrophecms/asset/lib/globalIcons.js

```diff
@@ -1,6 +1,7 @@
 // Alphabetical collection of globally available vue-material-design-icons.
 module.exports = {
   'account-box-icon': 'AccountBox',
+  'account-group-icon': 'AccountGroup',
```

Comment:
Do not add things to `globalIcons` in core in order to enable them in external npm modules. Instead, use the `icons` cascade in the module in question, which has the same effect.